app/Http/Controllers/TaskController.php

```diff
@@ -135,4 +135,37 @@ public function sort(Request $request)
         return $collectionTasks;
     }
 
+    public function getArchivedTasks(Request $request)
```

Comment:
This method is getting all tasks and not archived tasks. Your issue is to Create an API to fetch all tasks that have been archived.  What you have done here won't serve the purpose.